src/index.html

```diff
@@ -15,8 +16,103 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
   </head>
+  <!--#endregion -->
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        href="./index.html"
+        class="logo__image"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo_image"
+        />
+      </a>
+
+      <nav>
+        <ul class="nav__list">
+          <li class="nav__list__item">
+            <a
+              href="./index.html"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__list__item">
+            <a
+              href="./index.html"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__list__item">
+            <a
+              href="./index.html"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li
+            class="nav__list__item"
+            data-qa="hover"
+          >
+            <a
+              href="./index.html"
+              class="nav__link"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__list__item">
+            <a
+              href="./index.html"
+              class="nav__link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__list__item">
+            <a
+              href="./index.html"
+              class="nav__link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__list__item">
+            <a
+              href="./index.html"
```

Comment:
The `href` attribute for the navigation link is set to './index.html'. If this is a placeholder, please update it to the correct URL or use a relative path if it's meant for internal navigation. This applies to all navigation links.